src/object.d

```diff
@@ -1463,7 +1463,7 @@ const:
         assert(flag >= MItlsctor && flag <= MIname);
         assert(!(flag & (flag - 1)) && !(flag & ~(flag - 1) << 1));
     }
-    do
+    body
```

Comment:
Why? Use of `body` has been deprecated in favor of `do` with https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1003.md